tests/test_integration.py

```diff
@@ -1,23 +1,13 @@
-from unittest.case import SkipTest
-
-from hyper_sh import Client
-
-
-def assertSetup():
-    try:
-        Client.guess_config()
-    except RuntimeError:
-        raise SkipTest("no default config is detected")
```

Comment:
travis fails because you removed this skip. I dont think it is a good idea to put hyper creds in travis. so I only run those integ tests locally.

Reply:
That makes sense. I'll add that back.

Reply:
Tests are passing now.